pubsub/src/main/proto/spine/messaging/pubsub/pubsub_push.proto

```diff
@@ -0,0 +1,66 @@
+/*
```

Comment:
I am not sure why this file is called `pubsub_push.proto`. `pubsub` is present in both package name and library names. 

`push.proto` is enough.